packages/ui/src/hooks/use-social-callback-handler.ts

```diff
@@ -8,8 +8,14 @@ const useSocialCallbackHandler = () => {
 
   const socialCallbackHandler = useCallback(
     (connectorId: string) => {
-      // Apple use fragment mode to store auth parameter. Need to support it.
-      const search = window.location.search || '?' + window.location.hash.slice(1);
+      // Get search string to evaluate
+      const searchString = window.location.search;
+
+      // Get hash string to evaluate
+      const hashString = window.location.hash;
+
+      // Define evaluated search string
+      const search = `${searchString ?? '?'}${(searchString && hashString ? '&' : '')}${hashString.slice(1)}`;
```

Comment:
```suggestion
      const search = `${searchString || '?'}${
        searchString && hashString ? '&' : ''
      }${hashString.slice(1)}`;
```

Reply:
Some lint error remains. Not able to commit the change directly. Do you mind committing the updates thx. PR approved